build_manifest.json

```diff
@@ -210,27 +240,60 @@
     "buildDir": "yarn-project",
     "projectDir": "yarn-project/aztec.js",
     "dockerfile": "aztec.js/Dockerfile",
-    "rebuildPatterns": ["^yarn-project/aztec.js/"],
-    "dependencies": ["circuits.js", "foundation", "types"]
+    "rebuildPatterns": [
+      "^yarn-project/aztec.js/"
+    ],
+    "dependencies": [
+      "circuits.js",
+      "foundation",
+      "types"
+    ]
+  },
+  "canary-build": {
+    "buildDir": "yarn-project",
+    "projectDir": "yarn-project/canary",
+    "dockerfile": "canary/Dockerfile.build",
+    "rebuildPatterns": [
+      "^yarn-project/canary/"
+    ],
+    "dependencies": [
+      "aztec.js",
+      "l1-artifacts",
+      "noir-contracts"
+    ]
   },
   "canary": {
-    "buildDir": "canary",
-    "dockerfile": "Dockerfile",
-    "rebuildPatterns": ["^canary/"],
-    "dependencies": []
+    "buildDir": "yarn-project",
+    "projectDir": "yarn-project/canary",
+    "dockerfile": "canary/Dockerfile",
+    "rebuildPatterns": [
+      "^yarn-project/canary/"
+    ],
+    "dependencies": [
+      "aztec.js",
+      "l1-artifacts",
+      "noir-contracts"
+    ]
```

Comment:
Shouldn't the canary depend on aztec-rpc or aztec-node as well? Or is that covered as a transitive dependency?

Reply:
No, it's designed to depend solely on the packages that an external dev would use. Everything else is encapsulated within the sandbox.

Reply:
Yes I see what you mean. I forget we decoupled aztec.js from all of those other packages.